web/src/components/SpeakerCard.vue

```diff
@@ -86,7 +80,19 @@ export default {
       }
     },
   },
+  mounted() {
+    this.$getSpeakerTopics(this.speaker.id, this.setTopics, this.setError);
+  },
   methods: {
+    setTopics(records) {
+      this.topics = records.map((topic) => ({
+        id: topic.id,
+        text: topic.get('name'),
+      }));
+    },
+    setError(err) {
+      this.error = err;
+    },
```

Comment:
Following the previous comment, once you have the list of topics as props here on `SpeakerCard`, then you can just lookup the speaker topic IDs on the topic list here, as a computed property (here's the [example with prefectures](https://github.com/WWCodeTokyo/speak-her-db/blob/master/web/src/components/SpeakerCard.vue#L62-L65))

Reply:
Thanks for the great notes